atlite/datasets/era5.py

```diff
@@ -27,12 +28,6 @@
 import logging
 logger = logging.getLogger(__name__)
 
-import urllib3
-logger.warning('Disable urllib3 and cdsapi warnings.')
-urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
-logging.getLogger("cdsapi").setLevel(logging.ERROR)
```

Comment:
What was the reason for disabling those?

Reply:
I was annoyed by the continuous request info and `urllib3` spit out a lot of warning, but apparently the latter was caused by a wrong verification setup in my personal properties

Reply:
I think it would be nice to add two more logging statements of what is requested and what is downloaded to the `retrieve` function and then set the "cdsapi" log level to `logging.WARNING`.